src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -48,7 +48,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
-    <PackageReference Include="MlNetMklDeps" Version="$(MlNetMklDepsPackageVersion)" />
```

Comment:
I think you need to add this line to HalLearners project. Otherwise we don't fetch MKL at all and SymSGD can't be build.

Reply:
thanks Ivan. 

---
In reply to: [246568063](https://github.com/dotnet/machinelearning/pull/2091#discussion_r246568063) [](ancestors = 246568063)